src/Symfony/Component/HttpKernel/Tests/Fixtures/AcmeFooBundle/AcmeFooBundle.php

```diff
@@ -31,7 +31,7 @@ public function configure(DefinitionConfigurator $definition): void
 
     public function prependExtension(ContainerConfigurator $container, ContainerBuilder $builder): void
     {
-        $container->extension('loaded', ['bar' => 'baz'], true);
+        $container->extension('loaded', ['bar' => 'baz']);
```

Comment:
I read the previous discussion, but if we don't pass the prepend `true` we are not prepending that config, so I suggest use `$builder->prepend*` instead, or just remove the `prependExtension()` altogether if it's no longer tested or used.

Reply:
I think this is a better solution indeed. We're testing the `MergeExtensionConfigurationPass` so it makes sense to me to have as much assertions as possible on these cases, including on `prependExtension()`.